src/util/Constants.js

```diff
@@ -818,6 +818,16 @@ exports.InteractionResponseTypes = createEnum([
   'DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE',
 ]);
 
+/**
+ * NSFW level of a Guild
+ * * DEFAULT
+ * * EXPLICIT
+ * * SAFE
+ * * AGE_RESTRICTED
+ * @typedef {string} NsfwLevel
```

Comment:
```suggestion
 * @typedef {string} NSFWLevel
```

Reply:
Fixed in the latest commit. I should have waited a little. 😸